### ISSUE TITLE
Feature Request: Ability to configure distance metric system (ie Miles vs Kilometers/ Feet vs Meters)

### ISSUE THREAD

**Issue by shivthakker** [fleetbase/fleetbase#329]
Is your feature request related to a problem? Please describe.
The system currently supports KM.

Describe the solution you'd like
Add a setting to toggle between kilometers and miles.

Additional context
This feature would improve usability in regions where miles are the standard unit. (US, UK)

**Comment by d3cline** [comment]
100% a deal breaker, i had this software suite sold and yall lost the contract because of this, should have been done a long time ago. terrible.

**Comment by roncodes** [comment]
Understood, but only three countries in the world use miles as distance metric. Majority of our users and onboards are from outside the U.S. which is over 5,000 at least to this point. That said we are picking up more U.S. based companies recently and this feature will be added in an upcoming release as it's now more in demand.

**Comment by d3cline** [comment]
Yeah but one of those countries is the USA, your answer sucks and lost you a ton of money, period.

**Comment by roncodes** [comment]
Added in v0.7.5
